doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -137,28 +164,29 @@ You can download latest Service OS kernel from
 1. The latest Service OS kernel from the latest Clear Linux release
    from this area:
    https://download.clearlinux.org/releases/current/clear/x86_64/os/Packages.  Look for an
-   ``.rpm`` file named ``linux-pk414-sos-<kernel-version>-<build-version>.x86_64.rpm``.
+   ``.rpm`` file named ``linux-iot-lts2018-sos-<kernel-version>-<build-version>.x86_64.rpm``.
 
    While we recommend using the "current" (latest) release of Clear Linux, you can download
    a specific Clear Linux release from an area with that release number, e.g.: 
-   https://download.clearlinux.org/releases/24040/clear/x86_64/os/linux-pk414-sos-4.14.57-69.x86_64.rpm
+   https://download.clearlinux.org/releases/26440/clear/x86_64/os/Packages/linux-iot-lts2018-sos-4.19.0-22.x86_64.rpm
 
-#. Download and extract the latest Service OS kernel(this guide is based on 24040 as the current example)
+#. Download and extract the latest Service OS kernel(this guide is based on 26440 as the current example)
 
    .. code-block:: none
 
-      mkdir ~/kernel-build
-      cd ~/kernel-build
-      wget  https://download.clearlinux.org/releases/24040/clear/x86_64/os/Packages/linux-pk414-sos-4.14.57-69.x86_64.rpm
+      mkdir ~/sos-kernel-build
+      cd ~/sos-kernel-build
+      wget https://download.clearlinux.org/releases/26440/clear/x86_64/os/Packages/linux-iot-lts2018-sos-4.19.0-22.x86_64.rpm
       sudo apt-get install rpm2cpio
-      rpm2cpio linux-pk414-sos-4.14.57-69.x86_64.rpm | cpio -idmv
+      rpm2cpio linux-iot-lts2018-sos-4.19.0-22.x86_64.rpm | cpio -idmv
 
 #. Install the SOS kernel and its drivers (modules)
 
    .. code-block:: none
 
-      sudo cp -r ~/kernel-build/usr/lib/modules/4.14.57-69.pk414-sos/ /lib/modules/
-      sudo cp ~/kernel-build/usr/lib/kernel/org.clearlinux.pk414-sos.4.14.57-69 /boot/acrn/
+      sudo cp -r ~/sos-kernel-build/usr/lib/modules/4.19.0-22.iot-lts2018-sos/ /lib/modules/
+      mkdir /boot/acrn/
```

Comment:
I have a doubt in my mind that the ESP partition is mounted under `/boot/efi`. If true, does the step above work as expected or should the kernel be put somewhere on the ESP partition (i.e. under `/boot/efi`)? I don't have an Ubuntu system at hand to verify this so if someone can confirm this, it would be appreciated.